variants/rak4631/platformio.ini

```diff
@@ -20,33 +20,99 @@ lib_deps =
 debug_tool = jlink
 
 
+
 ; If not set we will default to uploading over serial (first it forces bootloader entry by talking 1200bps to cdcacm)
 ; Note: as of 6/2013 the serial/bootloader based programming takes approximately 30 seconds
 ;upload_protocol = jlink
 
 ; Allows programming and debug via the RAK NanoDAP as the default debugger tool for the RAK4631 (it is only $10!)
 ; programming time is about the same as the bootloader version.
 ; For information on this see the meshtastic developers documentation for "Development on the NRF52"
-[env:rak4631_dap]
+[env:rak4631_dbg]
 extends = env:rak4631
-board_level = extra
-; pyocd pack --i nrf52840
+
+; if the builtin version of openocd has a buggy version of semihosting, so use the external version
```

Comment:
board_level = extra should stay in.

Reply:
Oh thanks!  I botched that merge - will fix.

Reply:
@geeksville once the `board_level = extra` is back in, we can safely merge this I think

Reply:
@thebentern done!  thanks!